Sources/Components/Profile/IdentityView.swift

```diff
@@ -75,13 +86,42 @@ public class IdentityView: UIView {
         return imageView
     }()
 
+    private lazy var profileNameWrapperView: UIView = {
+        let view = UIView(withAutoLayout: true)
+
+        view.addSubview(profileNameLabel)
+        view.addSubview(verifiedBadge)
+
+        NSLayoutConstraint.activate([
+            profileNameLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            profileNameLabel.topAnchor.constraint(equalTo: view.topAnchor),
+            profileNameLabel.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+
+            verifiedBadge.leadingAnchor.constraint(equalTo: profileNameLabel.trailingAnchor, constant: .smallSpacing),
+            verifiedBadge.centerYAnchor.constraint(equalTo: profileNameLabel.centerYAnchor),
+            verifiedBadge.trailingAnchor.constraint(lessThanOrEqualTo: view.trailingAnchor),
+            verifiedBadge.widthAnchor.constraint(equalToConstant: 18),
+            verifiedBadge.heightAnchor.constraint(equalToConstant: 18)
```

Comment:
Again, not sure, but would be nice to have this in a method called `setupProfileViewWrapper` or something. 
Just to make it more convenient. It's up to you.